src/main/java/org/jenkinsci/plugins/github/pullrequest/builders/GitHubPRStatusBuilder.java

```diff
@@ -25,7 +28,9 @@
 /**
  * Sets pr status for build caused by GitHubPRCause
  */
-public class GitHubPRStatusBuilder extends Builder {
+public class GitHubPRStatusBuilder extends Builder implements SimpleBuildStep {
+    private static final long serialVersionUID = 1L;
```

Comment:
Mm... i saw plugins with Serialisable Builders... Just wanted to make it UID fixed in case of future changes.